docker/dev/rebuild.sh

```diff
@@ -0,0 +1,4 @@
+$CMAKE --build $BASE_PATH/parflow/build
```

Comment:
This script simply runs the make build as well as the python steps necessary to get pftools working. Meant for use within the container